protostar/commands/test_cairo1/test_cairo1_command.py

```diff
@@ -0,0 +1,139 @@
+from argparse import Namespace
```

Comment:
as discussed offline, I think we should move this to a separate folder like `protostar/commands/cairo1_commands` in the same manner as in  https://github.com/software-mansion/protostar/pull/1551. I'm open with using a different name, but I think it should be in one, separate directory